apiserver/Makefile

```diff
@@ -177,84 +164,31 @@ hack-lib:
-	  -e KUBECONFIG=/home/user/certs/kubeconfig \
-	  $(CALICO_BUILD) kube-apiserver \
-	    --etcd-servers=http://$(LOCAL_IP_ENV):2379 \
-	    --service-cluster-ip-range=10.101.0.0/16 \
-            --authorization-mode=RBAC \
-            --service-account-key-file=/home/user/certs/service-account.pem \
-            --service-account-signing-key-file=/home/user/certs/service-account-key.pem \
-            --service-account-issuer=https://localhost:443 \
-            --api-audiences=kubernetes.default \
-            --client-ca-file=/home/user/certs/ca.pem \
-            --tls-cert-file=/home/user/certs/kubernetes.pem \
-            --tls-private-key-file=/home/user/certs/kubernetes-key.pem \
-            --enable-priority-and-fairness=false \
-            --max-mutating-requests-inflight=0 \
-            --max-requests-inflight=0
-
-	# Wait until we can configure a cluster role binding which allows anonymous auth.
-	while ! docker exec st-apiserver kubectl create clusterrolebinding anonymous-admin --clusterrole=cluster-admin --user=system:anonymous; do echo "Trying to create ClusterRoleBinding"; sleep 2; done
-
-	# And run the controller manager.
-	docker run --detach --net=host \
-	  --name st-controller-manager \
-	  -v $(CURDIR)/test/certs:/home/user/certs \
-	  $(CALICO_BUILD) kube-controller-manager \
-            --master=https://127.0.0.1:6443 \
-            --kubeconfig=/home/user/certs/kube-controller-manager.kubeconfig \
-            --min-resync-period=3m \
-            --allocate-node-cidrs=true \
-            --cluster-cidr=192.168.0.0/16 \
-            --v=5 \
-            --service-account-private-key-file=/home/user/certs/service-account-key.pem \
-            --root-ca-file=/home/user/certs/ca.pem
-
-	# Create CustomResourceDefinition (CRD) for Calico resources
-	# from the manifest crds.yaml
-	docker run \
-		--net=host \
-		--rm \
-		-v $(CURDIR)/../:/go/src/github.com/projectcalico/calico:rw \
-		-v $(CURDIR)/test/certs:/home/user/certs \
-		lachlanevenson/k8s-kubectl:${KUBECTL_VERSION} \
-		--kubeconfig=/home/user/certs/kubeconfig \
-		apply -f /go/src/github.com/projectcalico/calico/libcalico-go/config/crd/
-
+run-kubernetes-server: run-k8s-controller-manager
+	# TODO: Are these actually needed?
 	# Create a Node in the API for the tests to use.
-	docker run \
-		--net=host \
-		--rm \
-		-v  $(CURDIR):/manifests \
-		-v $(CURDIR)/test/certs:/home/user/certs \
-		lachlanevenson/k8s-kubectl:${KUBECTL_VERSION} \
-		--kubeconfig=/home/user/certs/kubeconfig \
-		apply -f /manifests/test/mock-node.yaml
-
-	# Create Namespaces required by namespaced Calico `NetworkPolicy`
-	# tests from the manifests namespaces.yaml.
-	docker run \
-		--net=host \
-		--rm \
-		-v  $(CURDIR):/manifests \
-		-v $(CURDIR)/test/certs:/home/user/certs \
-		lachlanevenson/k8s-kubectl:${KUBECTL_VERSION} \
-		--kubeconfig=/home/user/certs/kubeconfig \
-		apply -f /manifests/test/namespaces.yaml
+	# docker run \
```

Comment:
Just a reminder to myself to double check if these are needed or if they were just copy/paste fluff from other places.